.github/workflows/binaries.yml

```diff
@@ -0,0 +1,56 @@
+name: Build binaries
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        required: true
+        type: string
+  workflow_call:
+    inputs:
+      version:
+        required: true
+        type: string
+
+jobs:
+  binaries:
+    name: Build lodestar binaries
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            build: |
+              npx caxa -D -p "yarn install --production" --input . --output "lodestar" -- "{{caxa}}/node_modules/.bin/node" "--max-old-space-size=8192" "{{caxa}}/node_modules/.bin/lodestar"
+              rm -rf .github/ assets/ audit/ dashboards/ docs/ docker/ scripts/
```

Comment:
This does nothing, has to be removed before creating the binary, or via `-e` option of caxa.

Check .dockerignore for reference of what we could potentially exclude, I don't think saving a few kbs makes sense. Also assets are needed for custom explorer.